fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -58,9 +58,10 @@
 
 
 def segment_FOV(
-    column: np.ndarray,
+    x: np.ndarray,
```

Comment:
Could we make it such that x is always 4D? To be checked.

Reply:
We can also make the single channel image a 4D image by using np.expand_dims. Cellpose handles this correctly afterwards.

I'll also adapt the tests to expect 4D images now